lib/multiplexer/multiplexer.go

```diff
@@ -191,14 +178,15 @@ func (m *Mux) Serve() error {
 			go m.detectAndForward(conn)
 			continue
 		}
-		if m.isClosed() {
+		if utils.IsUseOfClosedNetworkError(err) {
```

Comment:
FYI there's also `utils.IsOKNetworkError` which handles a few more scenarios of when a connection is closed somewhat expectedly.